rafay/eks_config.go

```diff
@@ -251,13 +251,20 @@ type WellKnownPolicies struct {
 
 // InlineDocument map[string]interface{}
 type InlineDocument struct {
-	Version   string          `yaml:"Version,omitempty"`
-	Statement InlineStatement `yaml:"Statement,omitempty"`
+	Id        string            `yaml:"Id,omitempty" json:"Id,omitempty"`
+	Statement []InlineStatement `yaml:"Statement,omitempty" json:"Statement,omitempty"`
+	Version   string            `yaml:"Version,omitempty" json:"Version,omitempty"`
 }
 type InlineStatement struct {
-	Effect   string   `yaml:"Effect,omitempty"`
-	Action   []string `yaml:"Action,omitempty"`
-	Resource string   `yaml:"Resource,omitempty"`
```

Comment:
Resource is changed from string to interface{}

Reply:
@chaitanya-rafay Change is in rafay struct not terraform, so users wont encounter schema issues. The expand and flatten functions will handle these changes.

The reason schema was changes to support all formats of input.